src/rules/no-global-function-names/index.js

```diff
@@ -2,6 +2,7 @@
 
 const valueParser = require("postcss-value-parser");
 const { utils } = require("stylelint");
+const getAtRuleParams = require("stylelint/lib/utils/getAtRuleParams.cjs");
```

Comment:
We should try to avoid importing from `stylelint`'s utils since they can change in future stylelint versions and cause a breaking change.

Could we instead create a copy of `getAtRuleParams` in this library?

Reply:
Sure, done in [`febc4c6`](https://github.com/stylelint-scss/stylelint-scss/pull/988/commits/febc4c61667bcd71a429de0e3d8813743c07fa15).